docs/digital_preservation.rst

```diff
@@ -1,12 +1,12 @@
 Digital Preservation
 --------------------
 
-This package has a strong emphasis on digital preservation, as we want to use lossless JP2s as our preservation master files. It was developed with input from our digital preservation team.
+This package has a strong emphasis on digital preservation, as we want to use lossless JP2s as our preservation master files. It was developed with input from our `digital preservation team <http://www.dpoc.ac.uk>`_.
 
 By default it checks:
 
 - the JP2 is valid (using `jpylyzer`_)
-- the JP2 can be converted back into a TIFF
+- the JP2 can be converted back into a TIFF with no loss
```

Comment:
In this bit I'm trying to be explicit about what 'with no loss' means rather than just saying it. I've made some minor changes to make it clearer.